### PR TITLE
{AzureCXP} fixes Azure/azure-powershell#18577

### PR DESCRIPTION
**Pull request by navba-MSFT**
Fixes Azure/azure-powershell#18577

This documentation is misleading in that it suggests that there's a way to "delete" any management operation. This PR will clarify and suggest escalating to MS if operations are stuck.

### CODE REVIEW

**Comment by PRMerger-2** [comment]
@navba-MSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by navba-MSFT** [comment]
@ramakoni1 Could you please review this PR once you get a chance? Thanks in advance.

**Comment by VanMSFT** [comment]
@MashaMSFT - Hey Masha, can you help review and get the right resource to verify in Rama's absence?

**Comment by navba-MSFT** [comment]
Thanks @VanMSFT 

@MashaMSFT Could you please look into this once you get a chance ? Thanks in advance.

**Review by MashaMSFT — CHANGES_REQUESTED** [review]
modified the text slightly

**Comment by PRMerger10** [comment]
@MashaMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: azure-sql/database/troubleshoot-common-errors-issues.md [hunk 1]
```diff
@@ -35,8 +35,8 @@ The Azure infrastructure has the ability to dynamically reconfigure servers when
...
.../><br/>The service is busy processing multiple create or update requests for your subscription or server. Requests are currently blocked for resource optimization. Query [sys.dm_operation_status](/sql/relational-databases/system-dynamic-management-views/sys-dm-operation-status-azure-sql-database) for pending operations. Wait until pending create or update requests are complete or delete one of your pending requests and retry your request later. If your operations seem to be stuck, [open a support ticket with Microsoft](/sql/sql-server/sql-server-get-help#contact-microsoft-support). For more information about resource limits, review: <br/>&bull; &nbsp;[Logical server resource limits](resource-limits-logical-server.md)<br/>&bull; &nbsp;[DTU-based limits for single databases](service-tiers-dtu.md)<br/>&bull; &nbsp;[DTU-based limits for elastic pools](resource-limits-dtu-elastic-pools.md)<br/>&bull; &nbsp;[vCore-based limits for single databases](resource-limits-vcore-single-databases.md)<br/>&bull; &nbsp;[vCore-based limits for elastic pools](resource-limits-vcore-elastic-pools.md)<br/>&bull; &nbsp;[Azure SQL Managed Instance resource limits](../managed-instance/resource-limits.md). |
```
Comment (MashaMSFT) [inline azure-sql/database/troubleshoot-common-errors-issues.md:38]:
```suggestion
| 49919 |16 |Cannot process create or update request. Too many create or update operations in progress for subscription "%ld".<br/><br/>The service is busy processing multiple create or update requests for your subscription or server. Requests are currently blocked for resource optimization. Query [sys.dm_operation_status](/sql/relational-databases/system-dynamic-management-views/sys-dm-operation-status-azure-sql-database) for pending operations. Wait until pending create or update requests are complete or delete one of your pending requests and retry your request later. If your operations seem to be stuck, wait for other in-progress operations to complete, or cancel them when possible. For example, you may be able to cancel a database copy or geo-replica creation by deleting the database or replica being created. If unable to cancel an apparently stuck operation, [open a support ticket with Microsoft](/sql/sql-server/sql-server-get-help#contact-microsoft-support).  |
```

File: azure-sql/database/troubleshoot-common-errors-issues.md [hunk 2]
```diff
@@ -35,8 +35,8 @@ The Azure infrastructure has the ability to dynamically reconfigure servers when
...
...ations in progress for subscription "%ld".<br/><br/>The service is busy processing multiple requests for this subscription. Requests are currently blocked for resource optimization. Query [sys.dm_operation_status](/sql/relational-databases/system-dynamic-management-views/sys-dm-operation-status-azure-sql-database) for operation status. Wait until pending requests are complete or delete one of your pending requests and retry your request later.  If your operations seem to be stuck, [open a support ticket with Microsoft](/sql/sql-server/sql-server-get-help#contact-microsoft-support). For more information about resource limits, review: <br/>&bull; &nbsp;[Logical server resource limits](resource-limits-logical-server.md)<br/>&bull; &nbsp;[DTU-based limits for single databases](service-tiers-dtu.md)<br/>&bull; &nbsp;[DTU-based limits for elastic pools](resource-limits-dtu-elastic-pools.md)<br/>&bull; &nbsp;[vCore-based limits for single databases](resource-limits-vcore-single-databases.md)<br/>&bull; &nbsp;[vCore-based limits for elastic pools](resource-limits-vcore-elastic-pools.md)<br/>&bull; &nbsp;[Azure SQL Managed Instance resource limits](../managed-instance/resource-limits.md). |
```
Comment (MashaMSFT) [inline azure-sql/database/troubleshoot-common-errors-issues.md:39]:
```suggestion
| 49920 |16 |Cannot process request. Too many operations in progress for subscription "%ld".<br/><br/>The service is busy processing multiple requests for this subscription. Requests are currently blocked for resource optimization. Query [sys.dm_operation_status](/sql/relational-databases/system-dynamic-management-views/sys-dm-operation-status-azure-sql-database) for operation status. Wait until pending requests are complete or delete one of your pending requests and retry your request later. If your operations seem to be stuck, wait for other in-progress operations to complete, or cancel them when possible. For example, you may be able to cancel a database copy or geo-replica creation by deleting the database or replica being created. If unable to cancel an apparently stuck operation, [open a support ticket with Microsoft](/sql/sql-server/sql-server-get-help#contact-microsoft-support). |
```

**Review by MashaMSFT — CHANGES_REQUESTED** [review]
modifying text

**Comment by PRMerger6** [comment]
@MashaMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by MashaMSFT** [comment]
#sign-off @ShannonLeavitt  @ktoliver fyi

**Pull request merged**